src/main/twirl/com/lightbend/lagom/docs/getstartedjavasbt.scala.html

```diff
@@ -39,7 +41,7 @@
                                 Change into the new directory and enter the following:
                             </p>
 
-                            <pre class="prettyprint"><code class="language-sh">sbt new lagom/lagom-java.g8</code></pre>
+                            <pre class="prettyprint"><code class="language-sh">sbt new -Dsbt.version=0.13.13 lagom/lagom-scala.g8</code></pre>
```

Comment:
Hi @rstento I think copy pasting here caused the command to replace `lagom/lagomljava.g8` with `lagom/lagom-scala.g8`.